c-cannons/bangazon-initial_site
# Ml general styling

#### Descriptions

Front page shows last 20 products added, and the user can now edit their profile.

#### Steps to Test

1. Products show on home page. (You can verify that they are the last 20 by looking in DB Browser)

2. Go to profile, edit profile

3. Change something and save. Verify the changes were saved.

#### Link to Feature Ticket

#18 
#17
